diff --git a/pipenv/vendor/click_completion/__init__.py b/pipenv/vendor/click_completion/__init__.py
index 1443c8f2..adb73f18 100644
--- a/pipenv/vendor/click_completion/__init__.py
+++ b/pipenv/vendor/click_completion/__init__.py
@@ -6,7 +6,10 @@ from __future__ import print_function, absolute_import
 import six
 
 from click import ParamType
-from enum import Enum
+try:
+    from enum import Enum
+except ImportError:
+    from pipenv.vendor.enum import Enum
 
 from click_completion.core import completion_configuration, get_code, install, shells, resolve_ctx, get_choices, \
     startswith, Shell
